processor/transformprocessor/internal/metrics/functions_test.go

```diff
@@ -23,13 +23,13 @@ import (
 )
 
 func Test_DefaultFunctions(t *testing.T) {
```

Comment:
This tests if there are any unexpected values, but not if any are missing. 

Is there any percentage in also turning the test around?

Reply:
@kentquirk I think technically the test would pass incorrectly if `actual` contained the same function definition an exact amount of times to match the length of `expected` and the function name was in `expected`.  I could add more logic into the check, but it might be overcomplicating things.  The main goal of the test is to ensure the signal-specific `Functions` values uses `common.Functions`.